deployment/README.md

```diff
@@ -1,18 +1,15 @@
 # Deployment
 
-## Contents
-  * [Prerequisites](#prerequisites)
-  * [Tanzu Application Service](#tanzu-application-service)
-  * [Tanzu Kubernetes Grid (vSphere)](#tanzu-kubernetes-grid-vsphere)
-  * [Pivotal Web Services](#pivotal-web-services)
-  * [Cloud Foundry](#cloud-foundry)
-  * [Heroku](#heroku)
-  * [Configuration](#configuration)
-  * [Testing your deployment](#testing-your-deployment)
-
-## Prerequisites
 1. Download and extract the latest package from the [releases page](https://github.com/pivotal/postfacto/releases)
-2. Choose a name for your app, we'll refer to this as `app-name` from now on
+1. Choose a name for your app, we'll refer to this as `app-name` from now on
+1. [Configure your deployment](#configuration) by adding Google Auth, enabling analytics, increasing the session timeout or setting the environment variable to run without Redis (only for v4.3.0 or higher with Postgres)
```

Comment:
Does it make sense to move this whole section up, given its position in this list?

Reply:
What do you mean? Move the configuration section up above the specific platform instructions?

Reply:
^ If yes, I agree. Might moving the platform instructions down might also make them less tempting to click on

Reply:
Yes, exactly; to reflect the fact that point 3 is the configuration, point 4 is the platforms and point 5 is testing the deployments, the rest of the docs should follow that order to make it more intuitive to navigate.